vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/GridDetailsRowIT.java

```diff
@@ -50,6 +50,14 @@ public void gridTwoItemsSelectedWhenOpen() {
         assertElementHasButton(detailsElement.get(1), "Person 2");
     }
 
+    @Test
+    public void shouldNotThrowOnDetailsClick() {
```

Comment:
Not sure how valuable this test is. This kind of implies that someone would add back an exception, or that some other random error might occur at some point.

Reply:
Agreed, it's kind of an odd thing to test. This applies to pretty much every test in the repo that uses `checkLogsForErrors()`.

Reply:
Alright, just wanted to get your opinion on this